tldw_Server_API/app/api/v1/endpoints/prompt_studio/prompt_studio_websocket.py

```
@@ -226,6 +234,17 @@ def get_client_count(self) -> int:
         """Get number of unique clients connected."""
         return len(self.active_connections)
 
+    async def close_all(self, timeout_s: float | None = None) -> None:
```

Ruff (0.15.7)

[warning] Unused method argument: `timeout_s`

(ARG002)